ci.jsonnet

```diff
@@ -177,7 +177,7 @@ local part_definitions = {
       # Otherwise sulong will change runtime truffle version
       is_after_optional+:: ["$.use.sulong"],
       setup+: [
-        [          
+        [
```

Comment:
Nice, I've missed those. I have configure vscode to trim trailing spaces as well.